pkg/reconciler/jbsconfig/jbsconfig.go

```diff
@@ -631,18 +624,9 @@ func (r *ReconcilerJBSConfig) handleNoImageSecretFound(ctx context.Context, conf
 			binding.Namespace = config.Namespace
 			imageRegistry := config.ImageRegistry()
 			url := "https://"
-			if imageRegistry.Host == "" {
-				url += "quay.io"
-			} else {
-				url += imageRegistry.Host
-			}
+			url += imageRegistry.Host
```

Comment:
Moved the defaulting of `quay.io` to `ImageRegistry()` call to make it more easily reused.